doc/source/monitoring.rst

```diff
@@ -32,12 +164,166 @@ into ElasticSearch/InfluxDB from where it then can be visualised using Kibana/Gr
+
+3) Import Elasticsearch indices
+
+Before transferring messages from the message queue to Elasticsearch, indices need to be defined in Elasticsearch. This is a list of the message formats of Rucio.
+
+
+**Transfer events**::
+
+    {
+      created_at: when the message was created (yyyy-MM-dd HH:mm:ss.SSSSSS)
+      event_type: type of this event (transfer-submitted, transfer-submittion_failed, transfer-queued, transfer-failed, transfer-done)
+      payload: {
+        account: account submitting the request
+        activity: activity of the request
+        bytes: size of the transferred file (byte)
+        checksum-adler: checksum using adler algorithm
+        checksum-md5: checksum using md5 alrogithm
+        created_at: Time when the message was created (yyyy-MM-dd HH:mm:ss.SSSSSS)
+        dst-rse: destination rse
+        dst-type: type of destination rse (disk, tape)
+        dst-url: destination url of the transferred file
+        duration: duration of the transfer (second)
+        event_type: type of this event (transfer-submitted, transfer-submittion_failed, transfer-queued, transfer-failed, transfer-done)
+        file-size: same as bytes
+        guid: guid of the transfer
+        name: name of transferred file
+        previous-request-id: id of previous request
+        protocol: transfer protocol
+        reason: reason of the failure
+        request-id: id of this request
+        scope: scope of the transferred data
+        src-rse: source rse
+        src-type: type of source rse (disk, tape)
+        src-url: source file url
+        started_at: start time of the transfer
+        submitted_at: submittion time of the transfer
+        tool-id: id of the transfer tool in rucio (rucio-conveyor)
+        transfer-endpoint: endpoint holder of the transfer (fts)
+        transfer-id: uuid of this transfer
+        transfer-link: link of this transfer (in form of fts url)
+        transferred_at: done time of this transfer
+      }
+    }
+
+**Deletion events**::
+
+    {
+      created_at: when the message was created (yyyy-MM-dd HH:mm:ss.SSSSSS)
+      event_type: type of this event (deletion-done,deletion-failed)
+      payload: {
+        scope: scope of the deleted replica
+        name: name of the deleted replica
+        rse: rse holding the deleted replica
+        file-size: size of the file
+        bytes: size of the file
+        url: url of the file
+        duration: duration of the deletion
+        protocol: prococol used in the deletion
+        reason: reason of the failure
+      }
+    }
+
+The formats of them are defined in `rucio-transfer.json <https://github.com/feipengsy/rucio-monitoring/blob/master/kibana/es-index/rucio-transfer.json>`__ and `rucio_deletion.json <https://github.com/feipengsy/rucio-monitoring/blob/master/kibana/es-index/rucio_deletion.json>`__ which could be imported into Kibana. 
```

Comment:
For these it would be good to mention this somehow within our repository as well. I am just worried about consistency over time, if your repository changes.